src/apps/vpn/ui/navigator/navigationBar/VPNBottomNavigationBarButton.qml

```diff
@@ -17,6 +18,27 @@ VPNIconButton {
     accessibleName: ""
 
     onClicked: {
+        switch(_screen) {
+            case VPNNavigator.ScreenMessaging:
+                VPNGleanDeprecated.recordGleanEventWithExtraKeys("bottomNavigationBarClick", {"bar_button": "ScreenMessaging"});
+                Glean.sample.bottomNavigationBarClick.record({ bar_button: "ScreenMessaging" });
+                break;
+
+            case VPNNavigator.ScreenHome:
+                VPNGleanDeprecated.recordGleanEventWithExtraKeys("bottomNavigationBarClick", {"bar_button": "ScreenHome"});
+                Glean.sample.bottomNavigationBarClick.record({ bar_button: "ScreenHome" });
+                break;
+
+            case VPNNavigator.ScreenSettings:
+                VPNGleanDeprecated.recordGleanEventWithExtraKeys("bottomNavigationBarClick", {"bar_button": "ScreenSettings"});
```

Comment:
This is bad. This switch-stmt does not scale. What about if:
1. we make navigator flags a bitmap
2. we introduce a record-to-glean flag
3. in Navigator.cpp you check if the flag contains the record-to-glean, and you call `VPNGleanDeprecated...`.

In this way, if tomorrow we want to record screenFooBar, we can simply pass a flag.

Reply:
100% agreed, it's some horrible code. This PR is not in review shape, apologies that I didn't catch that it automatically tagged code owners, even though I tried to put in on draft immediately. 

I thought from my testing, doing it in `Navigator.cpp` just recorded an integer, which is still not what we were hoping for here - but I can double check.

Reply:
Confirmed:
```
"name": "bottom_navigation_bar_click",
      "timestamp": 2,
      "extra": {
        "bar_button": "10"
      }
```

That said, there has got to be a better way to handle this for C++ enums. I couldn't find it on a quick search before, will do some more research or ask around tomorrow.

Reply:
You have to convert them enum to string. See similar glean events. For instance the update steps.

Reply:
Thanks. I don't believe there is a way to do this on the QML side, right? I have it working on the C++ side - thanks for the tips.

(This PR still isn't quite ready for review - I'll make it clear when it's in decent shape.)